modules/affinity/src/main/java/org/apache/ignite/internal/affinity/Assignments.java

```diff
@@ -53,32 +55,36 @@ public class Assignments implements Serializable {
      */
     private final boolean force;
 
+    /** Time when the catalog version that the assignments were calculated against becomes active (i. e. available for use). */
+    private final HybridTimestamp timestamp;
```

Comment:
Curios how it'll be serialized, as an object or as long? I'm talking about writing to the MS and not messages here, because in messaging it should be optimised. Basically I's asking whether we should use long instead of HybridTimestamp here?